aplt/scenarios.py

```diff
@@ -57,30 +131,66 @@ def basic_forever(notif_delay=300, run_once=0):
         yield ack(channel_id=notif["channelID"], version=notif["version"])
         yield counter("notification.ack", 1)
         yield wait(notif_delay)
+
         if run_once:
+            yield unregister(reg["channelID"])
             yield disconnect()
             break
 
 
-def connect_and_idle_forever():
+def notification_forever_stored(qty_stored=1000, ttl=300, flood_delay=30,
+                                notif_delay=30, run_once=0):
+    """Connects, then repeats every delay interval:
+    1. register
+    2. send notifications x qty_stored (# of notifications to store)
+    3. wait for flood_delay (seconds)
+    4. receive notifications x qty_stored
+
+    Repeats forever.
+    """
```

Comment:
This won't consistently work right now, until #41 is addressed. Upcoming changes may cause messages to be delivered slightly out of order, so relying on the order as this does is a bit brittle.